lab-khalid/test/jacketTester.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+const request = require('superagent');
+const expect = require('chai').expect;
+require('../server.js');
+
+describe('Jacket routes', function(){
+  var jacket = null;
+
+  describe('POST: /api/jacket', function(){
+    it('should return a jacket', function(done){
+      request.post('localhost:8000/api/jacket')
```

Comment:
make sure that the port you are using here is the same one your server is running on. In your server file your port number was set to be whatever process.env.PORT is or 3000. This means that if for some reason your server doesn't come up on the port specified in your process.env.PORT your tests will fail because the fall back port is 3000 not 8000.